packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import type { MongoClientOptions } from 'mongodb';
+import {
+  spacing,
+  Label,
+  Description,
+  Link,
+  Table,
+  TableHeader,
+  Row,
+  Cell,
+  Button,
+  IconButton,
+  Icon,
+} from '@mongodb-js/compass-components';
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+
+import { editableUrlOptions, UrlOption } from '../../../utils/url-options';
+
+import UrlOptionsModal from './url-options-modal';
+
+const urlOptionsContainerStyles = css({
+  marginTop: spacing[3],
+  width: '70%',
+});
+
+const addUrlOptionsButtonStyles = css({
+  textAlign: 'center',
+  marginTop: spacing[3],
+});
+
+const optionValueStyles = css({
+  span: {
```

Comment:
Again a hacky way of styling in leafy. is there a better way? why does Leafy complicate many things?

Reply:
I think you're running into styling issues here mostly because leafygreen table is not really built for something like that. To avoid this direct tag styling you can wrap everything you pass to the Cell in another element and apply styling to it.

Reply:
Weirdly enough I'm kinda struggling to find where this span is coming from, at a quick glance doesn't seem like leafygreen wraps its content into it and I can't see us adding it there, hmm... 🤔

Reply:
I tried to wrap the contents in a div and style it, but leafygreen wraps it in a span (and me too, I was not able to figure out where this span comes from). And the span limits styling it properly.

Reply:
I reached out to leafy channel on slack and will update once i hear back